pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -10976,22 +10976,22 @@
       heading: "Why teach computer vision?"
       desc: "Empower students to solve real-world challenges, foster innovation, and explore AI's ethical implications in visual data."
       real_world:
-        heading: "Real-world applications"
+        title: "Real-world applications"
```

Comment:
I updated these string names to match what I did for a similar section in the `game_design_page` namespace.